.github/workflows/build.yml

```diff
@@ -11,41 +11,53 @@ on:
 
 jobs:
   verify-python:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, ubuntu-latest]
```

Comment:
Maybe use ubuntu-24.04 here instead to avoid failures once this version changes?

Reply:
changes ubuntu-latest to ubuntu-24.04 in commit 23